tests/e2e/specs/pageProductOverview_spec.js

```diff
@@ -0,0 +1,42 @@
+describe('Product overview page', () => {
+  before(() => {
+    cy.visit('/products/men');
+  });
+  it('Changes sorting settings', () => {
```

Comment:
Have you tried searching for a way to make this code shorter?

Reply:
If it cannot be shorter, there are some ways to make it more readable. For example, placing some new lines between cypress commands, to group those commands that pretend to assert a same thing. It is kind of a ways to group subtests.

Another option could be to extract potentially duplicated logic into a method that describes what you want to assert.